src/main/java/com/minecolonies/coremod/research/MultiplierModifierResearchEffect.java

```diff
@@ -15,16 +16,22 @@
      */
     private double effect;
 
+    /**
+     * The relative value of the effect, for display purposes.
+     */
+    private double relativeEffect;
```

Comment:
maybe displayValue is better

Reply:
Right now, both are fed into the translationtexts, with relativeEffects simply the first argument and absolute effects the second. They only translation that displays absolute values for English is fleeingspeedaddition (which applies a potion effect, where absolute values make more sense).

But displayEffect is probably more logical of a separation.